cmd/brew-license.rb

```diff
@@ -21,17 +21,19 @@
 require "formula"
 require "optparse"
 
+# use only SPDX identifiers -> https://spdx.org/licenses/
+# except when a package's license does not have an SPDX identifier
 licenses = {
```

Comment:
Suggestion: put this array into a new file (possibly one file per formula) which will make it easier in future to update this file.